src/cuda_eval.cpp

```diff
@@ -194,7 +197,7 @@ void jitc_cuda_assemble(ThreadState *ts, ScheduledGroup group,
                 fmt("    mad.wide.u32 %rd0, %r0, $a, %rd0;\n", v);
 
             if (vt != VarType::Bool) {
-                fmt("    $s$t $v, [%rd0];\n",
```

Comment:
Would the existing `$b` not address the problem in a more general way here?

Reply:
Yes, you're right..I've fixed that up.